### PR TITLE
AKO Operator 1.5.1

### PR DESCRIPTION
**Pull request by RiyaGaniga-avi**
Akoconfig and values.yaml changes for AKO Operator 1.5.1

### CODE REVIEW

**Review by sudswasavi — COMMENTED** [review]
hi @RiyaGaniga-avi don't see the change for the T1Lr in this PR. Is this missed or deliberately left out?

**Review by sudswasavi — APPROVED** [review]

**Comment by sudswasavi** [comment]
Sorry just saw it. Look good. Thx.

**Comment by sudswasavi** [comment]
build ako

File: ako-operator/helm/ako-operator/values.yaml [hunk 1]
```diff
@@ -89,6 +90,8 @@ resources:
     cpu: 100m
     memory: 200Mi
 
+podSecurityContext: {}
```
Comment (hemantavi) [inline ako-operator/helm/ako-operator/values.yaml:93]: We don't need this here. If this was meant for AKO, we already add it via the AKO operator code.

**Review by hemantavi — COMMENTED** [review]
@RiyaGaniga-avi I don't see the code changes for the updated parameters in the AKOConfig object. They should go in the configmap file. Are you planning to raise another PR for it?

**Pull request closed without merging**